### ISSUE TITLE
Fine tuning train loss zero

### ISSUE THREAD

**Issue by sanakhamekhem** [pytorch/fairseq#3753]
Hello,

I have performed the self-supervised training (ASR)  using my own dataset. For pretraining, it is quite pertinent like the following accuracy curve on validation set.
 
![acc](https://user-images.githubusercontent.com/15616524/127737737-74f24ee9-08a4-47ee-9b08-6480a7928954.png)

However, when I perform the finetuning, the train loss and valid loss is always **zero**, the wer is 100%. 
I have checked the transcriptions files and dict etc, they are OK. Please any suggestions???

Here the command of tuning:
****
CUDA_VISIBLE_DEVICES=1  python train.py \
    --distributed-world-size 1  "${out_manifest_path_tuning}" \
    --save-dir $dirfinetuning \
	--min-loss-scale=0.5 \
    --valid-subset valid \
    --no-epoch-checkpoints \
    --best-checkpoint-metric wer \
    --num-workers 4 \
    --max-update 13000 \
    --sentence-avg \
    --task audio_pretraining \
    --arch wav2vec_ctc \
    --w2v-path $dirPretrained/checkpoint_best.pt \
    --labels ltr \
    --apply-mask \
    --mask-selection static \
    --mask-other 0 \
    --mask-length 10 \
    --mask-prob 0.75 \
    --layerdrop 0.05 \
    --mask-channel-selection static \
    --mask-channel-other 0 \
    --mask-channel-length 64 \
    --mask-channel-prob 0.256 \
    --zero-infinity \
    --feature-grad-mult 0.0 \
    --freeze-finetune-updates 0 \
    --validate-after-updates 4 \
    --optimizer adam \
    --adam-betas '(0.9, 0.98)' \
    --adam-eps 1e-08 \
    --lr 1e-04 \
    --lr-scheduler tri_stage \
    --warmup-steps 1300 \
    --hold-steps 5200 \
    --decay-steps 6500 \
    --final-lr-scale 0.05 \
    --final-dropout 0.0 \
    --dropout 0.0 \
    --activation-dropout 0.1 \
    --criterion ctc \
    --attention-dropout 0.0 \
    --max-tokens 1280000 \
    --seed 2337 \
    --log-format json \
    --log-interval 500 \
	--min-sample-size 1100 \
	--reset-optimizer \
	--tensorboard-logdir $TENSORBOARD_LOGDIR_TUNING \
    --ddp-backend no_c10d

**The log of training :**

2021-07-31 11:57:45 | INFO | fairseq_cli.train | end of epoch 5 (average epoch stats below)
2021-07-31 11:57:45 | INFO | train | {"epoch": 5, "train_loss": "0", "train_ntokens": "22092", "train_nsentences": "510.083", "train_nll_loss": "0", "train_wps": "2232.2", "train_ups": "0.1", "train_wpb": "22092", "train_bsz": "510.1", "train_num_updates": "60", "train_lr": "5.56923e-06", "train_gnorm": "0", "train_train_wall": "7", "train_gb_free": "9.5", "train_wall": "586"}
2021-07-31 11:57:57 | INFO | fairseq.trainer | begin training epoch 6
2021-07-31 11:57:57 | INFO | fairseq_cli.train | Start iterating over samples
